src/content/features/add-profile-matches-elo-points.js

```diff
@@ -15,9 +15,11 @@ import {
   setFeatureAttribute
 } from '../helpers/dom-element'
 import { calculateRatingChange } from '../helpers/elo'
-import getMatchHistory from '../helpers/match-history'
+import { getIsFreeMember } from '../helpers/membership'
 
 const FEATURE_ATTRIBUTE = 'elo-points'
+const MATCHES_PER_PAGE = 30 // FACEIT defined. Check the website to make sure this is up to date.
+let nextPage = 0
```

Comment:
This must be reset to 0 at some place, otherwise the feature stops working when navigating and returning at a later point, unless the user refreshes the page, which is something we want to avoid.

Reply:
Good catch - will work on a fix